mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -159,7 +160,15 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 			COMMENT the main adversary responsible for this entity's death
 	METHOD method_23883 dropXp ()V
 	METHOD method_24518 isHolding (Lnet/minecraft/class_1792;)Z
+		COMMENT Checks if this entity is holding a certain item.
+		COMMENT
+		COMMENT <p>This checks both the entity's main and off hand.
+		ARG 1
```

Comment:
```suggestion
		ARG 1 item
```
https://travis-ci.com/github/FabricMC/yarn/builds/155178823#L424
This fails the build